pako/README.md

```diff
@@ -12,5 +12,24 @@ you can require the packaged library like so:
 
 ```clojure
 (ns application.core
-  (:require cljsjs.pako))
+  (:require cljsjs.pako :as pako))
```

Comment:
When requiring foreign libs using `:as` or `:refer` is not supported. Functionality will always be imported into the global namespace the same way as the regular library makes itself available. In most cases this means `js/jQuery`, `js/pako` etc.